### ISSUE TITLE
disable side menu items option

### ISSUE THREAD

**Issue by zbeekman** [ProLoser/Github-Omnibox#54]
The side menu items look terrible and are distracting with the new github layout. They should be fixed and an option to disable them should be available in the settings UI

**Comment by ProLoser** [comment]
Yeah something is buggy with the corrections from Douglas, I will see if I can take a look at it tonight.

**Comment by zbeekman** [comment]
thanks :sparkles:

**Comment by ProLoser** [comment]
Closing this due to new work on #57 @zbeekman do you mind checking out those designs and giving feedback? I can still of course completely disable this feature.

**Comment by ProLoser** [comment]
Actually, I'll leave this open so we can just add a checkbox to disable this feature (and others) entirely.
